_docs/install-and-upgrade.md

```diff
@@ -6,14 +6,27 @@ redirect_from:
   - /docs/getting_started/install.html
 ---
 
+## Installation Methods
+
+* [Debian/Ubuntu-based distros](#debian-and-ubuntu-based-distributions)
+* [Arch Linux](#arch-linux)
+* [NPM Releases](#from-npm-releases)
+* [Docker](#docker)
+* [Running from source](#running-from-source)
+* [Unofficial Install Methods](#unofficial-install-methods)
+* [Services packaging The Lounge](#services-packaging-the-lounge)
+
+---
+
+
 ## Debian and Ubuntu based distributions
 
-First, make sure [Node.js](https://nodejs.org/) v4 or more recent is installed
+First, make sure [Node.js](https://nodejs.org/) v6 or more recent is installed
```

Comment:
Make it 6.13: https://github.com/thelounge/thelounge/blob/master/package.json#L39